src/borg/testsuite/archiver.py

```diff
@@ -228,6 +228,8 @@ def tearDown(self):
         os.chdir(self._old_wd)
         # note: ignore_errors=True as workaround for issue #862
         shutil.rmtree(self.tmpdir, ignore_errors=True)
+        # destroy logging configuration
+        logging.Logger.manager.loggerDict.clear()
```

Comment:
state leakage in the test i split in two. --progress creates some logging configuration that is carried over between invocations (self.cmd) and tests themselves in the `logging` module.